cmd/minishift/cmd/start.go

```diff
@@ -171,7 +172,10 @@ func runStart(cmd *cobra.Command, args []string) {
 	// preflight checks (after start)
 	preflightChecksAfterStartingHost(hostVm.Driver)
 
-	//Adding active profile information to all instance config
+	// Persist instance kube config path to be used by other commands
+	minishiftConfig.InstanceConfig.KubeConfigPath = filepath.Join(constants.Minipath, "machines", constants.MachineName+"_kubeconfig")
```

Comment:
Shouldn't we do this in PersistentPreRun in root.go. I do not see the value changing for each command and once set it can be reused by other commands.

Reply:
Make sense :+1: